packages/request-client.js/src/http-data-access.ts

```diff
@@ -4,6 +4,9 @@ import axios, { AxiosRequestConfig } from 'axios';
 
 import { EventEmitter } from 'events';
 
+const packageJson = require('../package.json');
+const REQUEST_CLIENT_VERSION_HEADER = 'X-Request-Client-Version';
```

Comment:
`X-Request-Client-Version` is this name not too generic?
maybe `X-Request-Network-Client-Version` to avoid collisions. 
It's a genuine question maybe the name is already specific enough.

Reply:
Good remark